tests/python/gaia-ui-tests/gaiatest/tests/functional/bzlite/test_search_bug.py

```diff
@@ -0,0 +1,13 @@
+from gaiatest import GaiaTestCase
+from gaiatest.apps.bzlite.app import BugzillaLite
+
+class TestSearchBug(GaiaTestCase):
+	def setUp(self):
+		GaiaTestCase.setUp(self)
+		self.connect_to_local_area_network()
+
+	def test_search_bug(self):
+		bugzilla_lite = BugzillaLite(self.marionette)
+		bugzilla_lite.launch()
+		bugzilla_lite.search(str(self.testvars['bugs']['5']))
```

Comment:
The fifth bug is [bug 127172](https://bugzilla.mozilla.org/show_bug.cgi?id=127172) which is not a Firefox OS bug. I think they filter out bugs that are not in Firefox OS. Is that correct @daleharvey ?